test/e2e/storage/testsuites/snapshottable.go

```diff
@@ -422,7 +422,7 @@ func (s *snapshottableTestSuite) DefineTests(driver storageframework.TestDriver,
 
 				ginkgo.By("modifying the data in the source PVC")
 
-				command := fmt.Sprintf("echo '%s' > %s", modifiedMntTestData, datapath)
+				command := fmt.Sprintf("echo '%s' > %s; sync", modifiedMntTestData, datapath)
```

Comment:
Does sync also have to be added in https://github.com/kubernetes/kubernetes/blob/227da7b0534d344a4a41ad2041b92a601e9765df/test/e2e/storage/testsuites/snapshottable.go#L145?

Reply:
good catch, added sync there too